flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontendParser.java

```diff
@@ -47,6 +47,11 @@
 			"Class with the program entry point (\"main\" method or \"getPlan()\" method. Only needed if the " +
 					"JAR file does not specify the class in its manifest.");
 
+	static final Option CLASSPATH_OPTION = new Option("C", "classpath", true, "Adds a URL to each user code classloader " +
```

Comment:
@tillrohrmann I added a comment that only protocols are supported which are supported by the `URLClassLoader`. The parameter above is Java standard.